src/main/kotlin/org/rust/lang/core/resolve/ref/RsMethodCallReferenceImpl.kt

```diff
@@ -31,7 +31,12 @@ class RsMethodCallReferenceImpl(
         val lookup = ImplLookup.relativeTo(element)
         val receiver = element.receiver.type
         return collectCompletionVariants {
-            processMethodCallExprResolveVariants(lookup, receiver, filterMethodCompletionVariants(it, lookup, receiver))
+            processMethodCallExprResolveVariants(lookup, receiver,
+                filterCompletionVariantsByVisibility(
+                    filterMethodCompletionVariants(it, lookup, receiver),
```

Comment:
Maybe now we should rename `filterMethodCompletionVariants` to something like `filterMethodCompletionVariantsByTraitBounds` ?